### PR TITLE
And in ships.txt

### PR DESCRIPTION
**Pull request by TheGiraffe3**
The plugin commit won't do anything if you squash it.

### CODE REVIEW

**Comment by Pilcrow182** [comment]
The change in the Partisan's description makes it harder to understand, imho. Perhaps the wording should be rearranged a bit?

> However, the removal of their armour plating makes them remarkably manueverable, especially when they have the latest atomic thrusters installed, at the cost of some of their hull integrity.

File: data/ships.txt [hunk 1]
```diff
@@ -1037,4 +1037,4 @@ ship "Partisan"
 	engine 14.5 30.5 0.5
 	explode "tiny explosion" 15
 	explode "small explosion" 5
-	description `When the Navy retires old fighters, they occasionally are sold on the civilian market as pleasure craft. These modified Lances, known as "Partisans," are not able to carry passengers like other pleasure craft, nor can they mount guns like a traditional Lance. However, the removal of their armour plating makes them remarkably manueverable at the cost of some of their hull integrity, especially with the latest atomic thrusters installed.`
+	description `When the Navy retires old fighters, they occasionally are sold on the civilian market as pleasure craft. These modified Lances, known as "Partisans," are not able to carry passengers like other pleasure craft, nor can they mount guns like a traditional Lance. However, the removal of their armour plating makes them remarkably manueverable at the cost of some of their hull integrity, especially being so when they have the latest atomic thrusters installed.`
```
Comment (TheGiraffe3) [inline data/ships.txt:1040]:
```suggestion
	description `When the Navy retires old fighters, they occasionally are sold on the civilian market as pleasure craft. These modified Lances, known as "Partisans," are not able to carry passengers like other pleasure craft, nor can they mount guns like a traditional Lance. However, the removal of their armour plating makes them remarkably manueverable, especially when they have the latest atomic thrusters installed, at the cost of some of their hull integrity.`
```

**Comment by TheGiraffe3** [comment]
@Pilcrow182 sorry, that was supposed to make you co-author the commit.

File: data/ships.txt [hunk 2]
```diff
@@ -1037,4 +1037,4 @@ ship "Partisan"
 	engine 14.5 30.5 0.5
 	explode "tiny explosion" 15
 	explode "small explosion" 5
-	description `When the Navy retires old fighters, they occasionally are sold on the civilian market as pleasure craft. These modified Lances, known as "Partisans," are not able to carry passengers like other pleasure craft, nor can they mount guns like a traditional Lance. However, the removal of their armour plating makes them remarkably manueverable at the cost of some of their hull integrity, especially with the latest atomic thrusters installed.`
+	description `When the Navy retires old fighters, they occasionally are sold on the civilian market as pleasure craft. These modified Lances, known as "Partisans," are not able to carry passengers like other pleasure craft, nor can they mount guns like a traditional Lance. However, the removal of their armour plating makes them remarkably manueverable, especially when they have the latest atomic thrusters installed, at the cost of some of their hull integrity.`
```
Comment (mOctave) [inline data/ships.txt:1040]: How about
```suggestion
	description `When the Navy retires old fighters, they occasionally are sold on the civilian market as pleasure craft. These modified Lances, known as "Partisans," are not able to carry passengers like other pleasure craft, nor can they mount guns like a traditional Lance. Paired with the removal of some of their armour plating and the addition of the latest atomic engines, this makes them exceptionally manueverable - albeit not very survivable in combat.`
```

**Pull request merged**